src/Azure/composer.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "open-telemetry/contrib-azure",
+    "description": "The Azure package for opentelemetry-php",
+    "keywords": ["opentelemetry", "otel", "azure", "tracing", "detector", "propagator"],
+    "type": "library",
+    "license": "Apache-2.0",
+    "authors": [
+        {
+            "name": "opentelemetry-php-contrib contributors",
+            "homepage": "https://github.com/open-telemetry/opentelemetry-php-contrib/graphs/contributors"
+        }
+    ],
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "php": "^7.4 || ^8.0",
+        "guzzlehttp/guzzle": "^7.8",
+        "open-telemetry/api": "^1.0",
+        "open-telemetry/sdk": "^1.0"
```

Comment:
ideally, and per [spec](https://opentelemetry.io/docs/concepts/instrumentation/libraries/#opentelemetry-api), instrumentation libraries should not depend on the SDK (which I think contradicts something I said in an earlier comment on this PR, sorry).

Reply:
Without the SDK dependency it's not possible to call `Registry::registerResourceDetector`. I modeled this after the container resource detector.

Is there an alternative approach you'd recommend?

Reply:
Yes, you are right. I was thinking that this was an "instrumenting library" but it's really an SDK extension. My bad, continue as you were :)